packages/grid/_modules_/grid/models/api/gridStateApi.ts

```diff
@@ -8,8 +8,7 @@ export interface GridStateApi {
   state: GridState;
   /**
    * Returns the state of the grid.
-   * @param {string} stateId The part of the state to be returned.
-   * @returns {any} The state of the grid.
+   * @returns {GridState} The state of the grid.
```

Comment:
Should we not document it for now?

```suggestion
   * @returns {GridState} The state of the grid.
   * @ignore - do not document.
```

Reply:
If we stop using it internally, we can just remove it.

Reply:
@flaviendelangle Removing the method would be a breaking change. We can move more iteratively.